### PR TITLE
feat: use monorepo queries

### PR DESCRIPTION
**Pull request by edgarkhanzadian**
> Try out Leather build 832434e — [Extension build](https://github.com/leather-wallet/extension/actions/runs/9222019695), [Test report](https://leather-wallet.github.io/playwright-reports/feat/use-monorepo-queries), [Storybook](https://feat-use-monorepo-queries--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=feat/use-monorepo-queries)

Connected to this PR in monorepo: https://github.com/leather-wallet/mono/pull/113

### CODE REVIEW

**Review by kyranjamie — APPROVED** [review]

**Comment by edgarkhanzadian** [comment]
This one is ready for review. Fix for failing tests is out for review in monorepo. Once that's merged I'll update the packages and it should fix the failing CI.

Other than that, codewise this PR is ready

**Comment by fbwoolf** [comment]
Sorry, I prob just really created a lot of conflicts for you with my last merge. If helps, I can rebase this for you?

**Comment by edgarkhanzadian** [comment]
@fbwoolf nope all good! Merged, thank you!

**Review by kyranjamie — APPROVED** [review]
Fantastic refactor work @edgarkhanzadian 👏🏼

File: tests/mocks/mock-inscriptions.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { Inscription } from '@shared/models/inscription.model';
+// import { createInscription } from '@leather-wallet/models';
```
Comment (fbwoolf) [inline tests/mocks/mock-inscriptions.ts]: Can these commented out lines be removed?

**Review by fbwoolf — APPROVED** [review]
🚀

File: src/app/app.tsx [hunk 2]
```diff
@@ -14,25 +18,48 @@ import { ToastsProvider } from '@app/features/toasts/toasts-provider';
...
+      <LeatherQueryProvider
+        client={queryClient}
+        network={network}
+        environment={{
+          env: WALLET_ENVIRONMENT,
+          github: {
+            org: GITHUB_ORG,
+            repo: GITHUB_REPO,
+            branchName: BRANCH_NAME,
+            localConfig: localConfig as any,
```
Comment (kyranjamie) [inline src/app/app.tsx:40]: We don't need to make this change now, but think the remote config should be separate

File: tests/mocks/mock-inscriptions.ts (hunk 1 above)
Comment (edgarkhanzadian) [reply tests/mocks/mock-inscriptions.ts]: Done!

File: src/app/app.tsx (hunk 2 above)
Comment (edgarkhanzadian) [reply src/app/app.tsx:40]: You mean like a separate package or it should be extension specific?

**Review by pete-watters — APPROVED** [review]
Great work 🚀

**Pull request merged**